src/patternfly/components/Card/card.scss

```diff
@@ -21,21 +22,49 @@
       box-shadow: var(--pf-c-card--m-hoverable--hover--BoxShadow);
     }
   }
+}
 
-  .pf-c-card__header,
-  .pf-c-card__body,
-  .pf-c-card__footer {
-    flex: 0 0 auto;
-    padding-right: var(--pf-c-card--child--PaddingRight);
-    padding-bottom: var(--pf-c-card--child--PaddingBottom);
-    padding-left: var(--pf-c-card--child--PaddingLeft);
+.pf-c-card__head {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
 
-    &:first-child {
-      padding-top: var(--pf-c-card--first-child--PaddingTop);
-    }
+  > :not(.pf-c-card__actions) {
+    flex: 1 1 auto;
+  }
+}
+
+.pf-c-card__actions {
+  display: flex;
+  align-items: center;
+  align-self: flex-start;
+  order: 99;
+  padding-left: var(--pf-c-card__actions--PaddingLeft);
+  margin-left: auto;
```

Comment:
if the `__head` element uses `justify-content: space-between`, do you need `margin-left: auto`? If you don't, I would suggest moving the padding value to be a margin instead.

Reply:
updated it so that if there is only one child, it gets margin-left: auto